nginx.conf

```diff
@@ -1,4 +1,5 @@
 server {
+    listen: 4000
```

Comment:
The nginx container crashes here. Looks like we need to ditch the colon and add a semicolon.

Reply:
Ack sorry about that.